cloudferry_devlab/cloudferry_devlab/tests/base.py

```diff
@@ -255,12 +255,20 @@ def get_volume_snapshot_id(self, snapshot_name):
         raise test_exceptions.NotFound('Snapshot with name "%s" was not found'
                                        % snapshot_name)
 
-    def get_user_tenant_roles(self, user):
+    def get_user_tenant_roles(self, user, tenant_attrib=None):
```

Comment:
As this function is called in two places and at both the places the behavior of this function was required to be changed because this function is supposed to validate the user tenant role for the passed migrated_tenant attrib or when there is no tenant attribute passed, it will behave like the original function which verifies the user tenant roles for all tenants.